omnigibson/robots/tiago.py

```diff
@@ -576,7 +576,13 @@ def set_position_orientation(self, position=None, orientation=None):
             position = current_position
         if orientation is None:
             orientation = current_orientation
-
+        
+        # Position and orientation are lists when restoring scene from json. Cast to np.array
+        if isinstance(position, list):
+            position = np.array(position)
+        if isinstance(orientation, list):
+            orientation = np.array(orientation)
```

Comment:
hmm I am not entirely sure why this would be happening. Either way, let's not fix this problem in `tiago.py` because this bug seems to be more general than the Tiago robot.

Reply:
Got it. I will revert these changes.